express-notes-api/server.ts

```diff
@@ -0,0 +1,73 @@
+import express, { Request, Response } from 'express';
+import fs from 'fs/promises';
+
+const app = express();
+
+type Note = {
+  nextId: number;
+  notes: Record<string, number>;
+};
+
+let notes: Note;
+
+async function updateNotes(): Promise<void> {
```

Comment:
`updateNotes` sounds like you are making a change to Notes. Instead, this function is reading the data. You should use a better function name.